cmd/datamon/cmd/bundle_mount.go

```diff
@@ -77,12 +78,23 @@ var mountBundleCmd = &cobra.Command{
 			return
 		}
 
-		path, err := sanitizePath(bundleOptions.DataPath)
-		if err != nil {
-			log.Fatalf("Failed to sanitize destination: %s\n", bundleOptions.DataPath)
-			return
+		var err error
+
+		var consumableStorePath string
+		if bundleOptions.DataPath == "" {
+			consumableStorePath, err = ioutil.TempDir("", "datamon-mount-destination")
```

Comment:
We can use the bundle hash and write the bundle descriptor last or write a done marker file. The next time we can skip downloading the files and reused the json files downloaded.. ?

Reply:
the bundle descriptor and filelists are written to disk at the same time it's used to initialize the in-memory structures.  #218 contains [the change](https://github.com/oneconcern/datamon/pull/218/files#diff-7d60c86cd0267ea4660946771a11f969R62) to read `${destination}/.datamon/` folder from disk into in-memory structures.

since this implementation places the destination in a temporary directory that might vanish after the process exits, it's not clear where the reusable json files are to be stored... i suppose creating a subdirectory of `${HOME}/.datamon`, say `${HOME}/.datamon/meta-cache`, is a reasonable option?

creating a new issue to document the intent of these changes seems more likely to me than including them in this diff.